internal/perms/perms_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// System permissions-related code unit tests.
+package perms_test
+
+import (
+	"syscall"
+	"testing"
+
+	"github.com/googlecloudplatform/gcsfuse/internal/perms"
+)
+
+func TestMyUserAndGroup(t *testing.T) {
```

Comment:
please refer to other test files to see how to write unit tests. We use jacobsa/ogletest framework for writing unit tests.